packages/image_picker/ios/Classes/ImagePickerPlugin.m

```diff
@@ -68,6 +68,11 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
     [alert addAction:library];
     [alert addAction:cancel];
     [_viewController presentViewController:alert animated:YES completion:nil];
+  }
+  else if([@"store" isEqualToString:call.method]) {
+    _result = result;
```

Comment:
I don't think we want to trample over `_result` here, we should be able to pass result to `storeFile:` as an additional argument and then to `UIImageWriteToSavedPhotosAlbum` as the `contextInfo` argument